src/language-css/printer-postcss.js

```diff
@@ -574,6 +573,9 @@ function genericPrint(path, options, print) {
 }
 
 const printer = {
+  experimentalFeatures: {
+    frontMatterSupport: true,
+  },
```

Comment:
If this is not an established pattern yet, might be worth calling the option `experimental`. It's shorter and avoids having plural for a lookup. Plural usually suggests that we've got an array of things we can iterate over.

`next.config.ts` has `experimental` too.

Reply:
I think it's easier for plugin, they only need remove the prefix later.

docusaurus  `experimental_faster` https://github.com/prettier/prettier/blob/e5118b26506c0f1d61ca122b9950615886005051/website/docusaurus.config.js#L279

typescript-eslint have `EXPERIMENTAL_useProjectService`